src/main/java/core/activities/ActivitiesHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.activities;
+
+public interface ActivitiesHandler {
+    int getAmountOfFruits(int fruitsInShop, int fruitsInCirculation);
```

Comment:
```suggestion
public interface AmountHandler {
    int calculateAmount(int currentAmount, int operationAmount);
```

let's try to create more abstract but informative names for strategy pattern